### PR TITLE
#773 remove deprecated istanbul config flags

### PR DESCRIPTION
**Pull request by jcortejoso**
### Description
Removed deprecated celo-blockchain cmd flags as described in celo-labs#773 from clabs testnet deployment code (helm charts and terraform modules), and from the documentation. This do not modify some internal tests using those cmd options.

### Tested
Validated the templates and chart against Kubernetes api.

### Related issues

- Fixes celo-labs/773

### Backwards compatibility
No Backwards compatibility problem

### CODE REVIEW

File: packages/terraform-modules-public/gcp/testnet/modules/validator/startup.sh [hunk 1]
```diff
@@ -429,6 +427,7 @@ ExecStart=/usr/bin/docker run \\
       --proxy.proxyenodeurlpair=\\"$PROXY_URL\\" \\
   "
 ExecStop=/usr/bin/docker stop -t 60 %N
+ExecStopPost=/usr/bin/docker rm -f %N
```
Comment (oneeman) [inline packages/terraform-modules-public/gcp/testnet/modules/validator/startup.sh:430]: What's this about?

**Review by oneeman — COMMENTED** [review]
Flag related changes look good.  Wasn't sure how `ExecStopPost` fits into this PR.

File: packages/terraform-modules-public/gcp/testnet/modules/validator/startup.sh (hunk 1 above)
Comment (jcortejoso) [reply packages/terraform-modules-public/gcp/testnet/modules/validator/startup.sh:430]: It is not related with this PR. It is a bug in the systemd unit that cause you cannot restart the service with systemd tools (systemd stop/restart, or if you restart the instance...) and the container needs to manually removed with dockercli to reconciliate with systemd.
Package `terraform-modules-public` is meant to be a template/skeleton for anyone to run infra connected to celo network.

**Review by oneeman — APPROVED** [review]
LGTM, approved.

**Pull request merged**